README.md

```diff
@@ -96,40 +88,26 @@ function.
 <img src="Images/sublevf.png" style="display: block; margin: auto;" />
 
 Above is an example for a function defined on a subset of
-![\\mathbb{R}](https://latex.codecogs.com/png.latex?%5Cmathbb%7BR%7D
-"\\mathbb{R}") but in general the function
-![f](https://latex.codecogs.com/png.latex?f "f") is defined on a subset
-of
-![\\mathbb{R}^d](https://latex.codecogs.com/png.latex?%5Cmathbb%7BR%7D%5Ed
-"\\mathbb{R}^d").
+$\mathbb{R}$ but in general the function $f$ is defined on a subset of
+$\mathbb{R}^d$.
 
 [Notebook: cubical complexes](Tuto-GUDHI-cubical-complexes.ipynb). One
 first approach for studying the topology of the sublevel sets of a
 function is to define a regular grid on
-![\\mathbb{R}^d](https://latex.codecogs.com/png.latex?%5Cmathbb%7BR%7D%5Ed
-"\\mathbb{R}^d") and then to define a filtered complex based on this
-grid and the function ![f](https://latex.codecogs.com/png.latex?f "f").
+$\mathbb{R}^d$ and then to define a filtered complex based on this grid and the
+function $f$.
 
 ### 02 - Persistent homology and persistence diagrams
 
 Homology is a well-known concept in algebraic topology. It provides a
 powerful tool to formalize and handle the notion of topological features
 of a topological space or of a simplicial complex in an algebraic way.
-For any dimension ![k](https://latex.codecogs.com/png.latex?k "k"), the
-![k](https://latex.codecogs.com/png.latex?k "k")-dimensional *holes* are
-represented by a vector space
-![H\_k](https://latex.codecogs.com/png.latex?H_k "H_k"), whose dimension
-is intuitively the number of such independent features. For example, the
-![0](https://latex.codecogs.com/png.latex?0 "0")-dimensional homology
-group ![H\_0](https://latex.codecogs.com/png.latex?H_0 "H_0") represents
-the connected components of the complex, the
-![1](https://latex.codecogs.com/png.latex?1 "1")-dimensional homology
-group ![H\_1](https://latex.codecogs.com/png.latex?H_1 "H_1") represents
-the ![1](https://latex.codecogs.com/png.latex?1 "1")-dimensional loops,
-the ![2](https://latex.codecogs.com/png.latex?2 "2")-dimensional
-homology group ![H\_2](https://latex.codecogs.com/png.latex?H_2 "H_2")
-represents the ![2](https://latex.codecogs.com/png.latex?2
-"2")-dimensional cavities and so on.
+For any dimension $k$, the $k$-dimensional *holes* are represented by a vector
+space $H\_k$, whose dimension is intuitively the number of such independent
+features. For example, the $0$-dimensional homology group $H\_0$ represents the
+connected components of the complex, the $1$-dimensional homology group $H\_1$
+represents the $1$-dimensional loops, the $2$-dimensional homology group $H\_2$
+represents the $2$-dimensional cavities and so on.
```

Comment:
I don't know if it is that useful to write `$2$` instead of just `2`, but if you like it that way, ok.

Reply:
I like the mathjax rendering for these ones and I just kept how Bertrand wrote it.
I let it like that.